src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,76 @@
+package duke;
+
+import duke.Gui.Gui;
+import duke.command.Command;
+import duke.component.DukeException;
+import duke.component.Parser;
+import duke.component.Storage;
+import duke.task.TaskList;
+import javafx.application.Application;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+public class Duke {
+    private static final Path filepath = Paths.get(".", "data", "duke.txt");
+    private Storage storage;
+    private TaskList taskList;
+    private boolean isExtracted = true;
+
+    /**
+     * Creates an instance of Duke, instance of ui created too. TaskList initialised.
```

Comment:
Can consider adopting a consistent grammatical style. So instead of saying `Creates an instance of Duke, instance of ui created too. TaskList initialised.` consider phrasing it as something along the lines of `Creates an instance of Duke and ui. Initialises TaskList`